app/javascript/v3/views/onboarding/OnboardingStep.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div
+    v-if="!hidden"
+    class="flex items-center gap-2 p-2 text-lg font-bold mb-6 relative before:absolute before:h-10 before:w-[1px] before:bg-slate-200 before:-bottom-8 before:left-[24px] hide-before-of-last"
+    :class="{
+      'text-woot-500 ': isActive,
+      'text-slate-400': (!isActive && !isComplete) || isComplete,
+      'before:bg-woot-500': !isActive && isComplete,
+      hidden: hidden,
+    }"
+  >
+    <div
+      class="grid w-8 h-8 border border-solid rounded-full place-content-center"
+      :class="{
+        'border-woot-500': (isActive && !isComplete) || isComplete,
```

Comment:
```suggestion
      'border-woot-500': !isActive || isComplete,
```

We can simplify the condition,  since if `isComplete` is true, the condition will always be true, regardless of `isActive`.